src/lib.rs

```diff
@@ -11,6 +11,19 @@
 //! `H(N,s)` is the normalizing constant which corresponds to the generalized harmonic number
 //! of order `N` of `s`.
 //!
+//!
+//! # Example
+//!
+//! ```
+//! extern crate rand;
+//! use rand::thread_rng;
```

Comment:
I would leave off this `use` and call `rand::thread_rng` directly in the code above. That way it's clear that the trait is the primary thing users need to import.